### PR TITLE
Implement Forestry CMS 🌲

### PR DESCRIPTION
**Pull request by deammer**
This PR implements [Forestry](https://forestry.io) as a content management system. Forestry is a hosted service that commits changes directly into the repo, so we maintain full control over our content.

My biggest gripe with other systems (like Contentful) is that every collaborator needs to download a bunch of content from the CMS (using an API key) before they're able to collaborate. With Forestry, there's only one source of truth, and collaborators don't require access to the editor UI.

Here's what the editor UI looks like. It feels very easy to use and would enable people without GitHub or dev experience to make edits easily.

Screen Shot 2022-01-13 at 10 29 12 PM

👉 What do y'all think? Honest feedback, please!

### CODE REVIEW

**Comment by deammer** [comment]
Please note: I haven't finished migrating everything to the new CMS. I wanted to get buy-in from people before spending time on migrating images and such.

File: content/site-settings.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+instagram_url: 'https://www.instagram.com/distributeaid'
```
Comment (deammer) [inline content/site-settings.yml:1]: This file contains global settings that can be applied on every page of the site.

File: content/site-settings.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+twitter_url: 'https://twitter.com/DistributeAid'
+linkedin_url: 'https://www.linkedin.com/company/distribute-aid'
+facebook_url: 'https://www.facebook.com/DistributeAidDotOrg/'
+footer_email: 'hello@distributeaid.org'
```
Comment (deammer) [inline content/site-settings.yml:5]: This is a test

**Comment by RabeaGleissner** [comment]
Nice! Thanks for this. Tbh, I'd never heard of Forestry CMS before but just had a little look and it sounds really good!! I'd be happy to go ahead with this. 

The only small thing I noticed in this PR is that you're sometimes using snake case in the yaml. Might be better to use camel case so it fits with the rest of the JS naming. I don't think yaml cares and ultimately most of the code is JS.

**Review by hola-soy-milk — APPROVED** [review]
This is super exciting! Reminds me a little of Netlify CMS, but open for us to use directly in git! 

I'm also excited about how everything is clear and editable in Github. Nice one!

**Comment by hola-soy-milk** [comment]
> Nice! Thanks for this. Tbh, I'd never heard of Forestry CMS before but just had a little look and it sounds really good!! I'd be happy to go ahead with this.
> 
> The only small thing I noticed in this PR is that you're sometimes using snake case in the yaml. Might be better to use camel case so it fits with the rest of the JS naming. I don't think yaml cares and ultimately most of the code is JS.

This is an excellent question! In my experience, YAML is usually formatted with snake_case, but I'm not super picky. I'll leave it to you, @deammer! Thanks again for looking into forestry!

Just had a chat with @khendrikse live on stream, who found that the consensus is that folks adapt their YAML casing to fit the project, so it would seem that using the project's camelCase would be a best practise, after all!

**Comment by hola-soy-milk** [comment]
Thanks again for this! @jtfairbank and I went ahead and added the Fort Pickett content. Super excited to start rolling out Forestry! 🚀

**Pull request merged**